GoodMorning/GoodMorning/UIImage+.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  UIImage+.swift
+//  GoodMorning
+//
+//  Created by Miro on 2023/08/08.
+//
+
+import UIKit
+
+extension UIImage {
+
+    static var checkmark: Self? {
+        let SFSymbolConfiguration = UIImage.SymbolConfiguration(
+            pointSize: 15,
+            weight: .bold,
+            scale: .large
+        )
+        let SFSymbolImage = UIImage(
+            systemName: "checkmark",
+            withConfiguration: SFSymbolConfiguration
+        )?.withTintColor(.black, renderingMode: .alwaysOriginal)
+
+        return SFSymbolImage as? Self
+    }
```

Comment:
`checkmark`의 타입을 Self가 아닌 UIImage로 설정해 준다면 return할 때 타입 캐스팅을 해주지 않아도 됩니다!

아, 그리고 조금 개인의 취향이지만,, 지금 `SymbolConfiguration`을 사용해 해당 UIImage에 대한 속성을 다 정의해 주었는데, 뒤에 붙은 `withTintColor`이 조금 어색해 보여요!
차라리 이 부분은 해당 `checkmark`를 사용하는 UIButton에서 tintColor를 정의해보는건 어떨까요? 
(하지만 이건 개인의 취향입니다!)

```suggestion
  static var checkmark: UIImage? {
        let SFSymbolConfiguration = UIImage.SymbolConfiguration(
            pointSize: 15,
            weight: .bold,
            scale: .large
        )
        let SFSymbolImage = UIImage(
            systemName: "checkmark",
            withConfiguration: SFSymbolConfiguration
        )

        return SFSymbolImage
    }
```

Reply:
넵! 써니가 해주신대로 구현하는 것이 훨씬 깔끔하고 좋네요! 좋은 피드백 감사합니다!~👍

> 커밋: e8e8cd8